serverless.yml

```diff
@@ -0,0 +1,50 @@
+service: poprox-default-recommender
```

Comment:
This configuration tells `serverless` how to set our code up as an AWS Lambda function. Most of it is boilerplate, but the interesting part is the `functions:` section at the bottom